infrastructure/src/main/java/org/corfudb/infrastructure/logreplication/LogicalGroupBasedReplicationConfig.java

```diff
@@ -0,0 +1,55 @@
+package org.corfudb.infrastructure.logreplication;
+
+import lombok.ToString;
+import lombok.extern.slf4j.Slf4j;
+import org.corfudb.infrastructure.ServerContext;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import java.util.UUID;
+
+@Slf4j
+@ToString
+public class LogicalGroupBasedReplicationConfig extends LogReplicationConfig {
```

Comment:
Hey Sundar, do we have some doc/confluence page that describes logical group based readers? or how it differs from the regular LR readers? If yes, could you please add it in the change description?

Reply:
hi Sourav, let me ping you offline.